client/my-sites/checkout/composite-checkout/components/item-variation-picker/variant-dropdown-price.tsx

```diff
@@ -95,6 +96,23 @@ export const ItemVariantDropDownPrice: FunctionComponent< {
 	const formattedCompareToPriceForVariantTerm = compareToPriceForVariantTerm
 		? formatCurrency( compareToPriceForVariantTerm, variant.currency, { stripZeros: true } )
 		: undefined;
+	// Jetpack introductory pricing displays the introductory price for the first term, then the regular price for the remaining term.
+	const planTerm = variant.termIntervalInDays > 31 ? 'year' : 'month';
```

Comment:
Values should be translatable since they're used as is in a string.

Though it's not a concern for Jetpack products at the moment, I'm wondering if this could be more future proof. WPcom plans have monthly, annual, and biennial terms I believe. It'd be great if we could support different terms.